### PR TITLE
#3920 Create nds text input

### PR DESCRIPTION
**Pull request by drakkein**

### CODE REVIEW

File: packages/web/app/styles/mixins/typography.scss [hunk 1]
```diff
@@ -151,6 +151,28 @@
   line-height: 1.71;
 }
 
+@mixin nds-input-text {
```
Comment (Moejoe90) [inline packages/web/app/styles/mixins/typography.scss]: are these new typography styles? if so are they in the style guide?

File: packages/web/app/components/shared/inputs/InputError.module.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.error-message {
+  @include nds-input-label-small;
+
+  color: $neu-flamingo-red;
+  text-transform: none;
+  margin: 4px 0;
```
Comment (Moejoe90) [inline packages/web/app/components/shared/inputs/InputError.module.scss]: can you make this `rem`

File: packages/web/app/components/shared/inputs/InputDescription.module.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.description {
+  @include nds-input-label-small;
+
+  color: $neu-gull-gray;
+  text-transform: none;
+  margin: 4px 0;
```
Comment (Moejoe90) [inline packages/web/app/components/shared/inputs/InputDescription.module.scss]: use `px-to-rem` please

File: packages/web/app/components/shared/inputs/Field.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  name: string;
+  label?: string;
+  description?: string;
+  isValid?: boolean;
+  required?: boolean;
+  disabled?: boolean;
+  placeholder?: string;
+};
+//TODO: connect to formik after update
+export const FieldLayout: React.FunctionComponent<TFieldProps &
```
Comment (Moejoe90) [inline packages/web/app/components/shared/inputs/Field.tsx]: Please don't wait for the update. 
We need to use this component in its connected Form use `FormInput.tsx` as a guide on how to use `FormikConsumer` etc.

File: packages/web/app/styles/mixins/typography.scss (hunk 1 above)
Comment (drakkein) [reply packages/web/app/styles/mixins/typography.scss]: These styles are missing from NDS Text styles catalog, but are present in inputs styleguide

Comment (Moejoe90) [reply packages/web/app/styles/mixins/typography.scss]: Then we need to talk with @andywalton about this

**Comment by drakkein** [comment]
@glupiawymowka Error message not displaying has been fixed, sorry for your lost time.

**Pull request merged**